sdk/storage/Azure.Storage.Blobs/src/QuickQueryExtensions.cs

```diff
@@ -57,6 +57,15 @@ internal static class QuickQueryExtensions
                 serialization.Format.ArrowConfiguration = new ArrowTextConfigurationInternal(
                     arrowConfiguration.Schema?.Select(ToArrowFieldInternal).ToList());
             }
+            else if (textConfiguration is BlobQueryParquetTextOptions parquetTextConfiguration)
+            {
+                if (!isInput)
+                {
+                    throw new ArgumentException($"{nameof(BlobQueryParquetTextOptions)} can only be used for input serialization.");
```

Comment:
this isn't covered by test.

Reply:
same for datalake .

Reply:
I think it's unnecessary for Data Lake.  Data Lake calls into the BlockBlobClient.Query()/QueryAsync(), and follows this exact code path that was covered by the test I just wrote for Blobs.

Reply:
Why does code path like this exist in sdk/storage/Azure.Storage.Files.DataLake/src/DataLakeExtensions.cs ?

Reply:
TODO write this test.

Reply:
fixed.